app/common/UserAPI.ts

```diff
@@ -1200,7 +1220,7 @@ export class DocAPIImpl extends BaseAPI implements DocAPI {
 
   public async uploadAttachment(value: string | Blob, filename?: string): Promise<number> {
     const formData = this.newFormData();
-    formData.append('upload', value as Blob, filename);
+    formData.append('upload', value, filename);
```

Comment:
Hmm, why we don't need it anymore? (Maybe it was needed in saas build).

Reply:
`FormData` objects natively accept `string | Blob` in `.append`. I just looked through the Git history, and couldn't see any obvious reason why this was added. Maybe the types accepted by `FormData.append` changed? 

Seems fine now either way!

Reply:
I will check in saas's version of typescript. I remember I was the one added it as the types were complaining.

Reply:
Great, let me know if it complains :)